static/bidder-info/lemmadigital.yaml

```diff
@@ -11,4 +11,8 @@ capabilities:
   site:
     mediaTypes:
       - banner
-      - video
\ No newline at end of file
+      - video
+userSync:
+  iframe:
+    url: "https://sync.lemmadigital.com/js/usersync.html?redirect={{.RedirectURL}}"
+    userMacro: "${UUID}"
```

Comment:
![Screenshot 2024-08-19 at 3 28 45 PM](https://github.com/user-attachments/assets/fa42a728-ac33-4223-b16a-c78c6f899c93)

Usersync url is not working.

Reply:
Here is an URL that I tried - https://sync.lemmadigital.com/js/usersync.html?redirect=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dlemmadigital%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%24%7BUUID%7D

Reply:
@imedvedko Requesting you to please fix this usersync URL.

Reply:
Looking into this

Reply:
> @imedvedko Requesting you to please fix this usersync URL.

It's not me 😅

Reply:
@gargcreation1992 , please check now, we've made an update to the url.

Reply:
UID is still not being set. Getting the following redirect URL - 
```
https://ib.adnxs.com/prebid/setuid?bidder=lemmadigital&gdpr=&gdpr_consent=&gpp=&gpp_sid=&f=b&uid=${UUID}&

Reply:
@gargcreation1992 can you share which URL you used to check this?
It worked for me:
![image](https://github.com/user-attachments/assets/686f5d54-38c4-4899-afc7-0cd77a275243)
URL used:
```
https://sync.lemmadigital.com/setuid?publisher=850&redirect=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dlemmadigital%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%24%7BUUID%7D
```

Reply:
@gargcreation1992 , did you get a chance to check this comment: https://github.com/prebid/prebid-server/pull/3862#discussion_r1732269887?
Share the details please if there is additional feedback.

Reply:
user sync verified using `https://sync.lemmadigital.com/setuid?publisher=850&redirect=http%3A%2F%2Flocalhost%3A8000%2Fsetuid%3Fbidder%3Dlemmadigital%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%24%7BUUID%7D`

<img width="835" alt="Screenshot 2024-09-13 at 3 22 41 PM" src="https://github.com/user-attachments/assets/d9417851-3e87-408c-9d7b-e451b90a6bf2">


cookie after sync completes:
```
echo "eyJ0ZW1wVUlEcyI6eyJsZW1tYWRpZ2l0YWwiOnsidWlkIjoiNDUxODEwOWMtNzIwMC0xMWVmLWFjNWUtZDA4ZTc5ZjZjZjdlIiwiZXhwaXJlcyI6IjIwMjQtMDktMjdUMTU6MjA6NDAuNDkzNDYxLTA0OjAwIn19fQ" | base64 -D
{"tempUIDs":{"lemmadigital":{"uid":"4518109c-7200-11ef-ac5e-d08e79f6cf7e","expires":"2024-09-27T15:20:40.493461-04:00"}}
```